src/config/networks.ts

```diff
@@ -136,7 +136,7 @@ const makeNetworkList = () => {
         },
       },
       subscanEndpoint: 'http://127.0.0.1:4399',
-      subscanUrl: 'https://creditcoin3-testnet.subscan.io/',
+      subscanUrl: 'https://creditcoin3-testnet.subscan.io',
```

Comment:
This need to matches whatever it's on the other networks, I think this one is fine. But just to remember double check it.